### PR TITLE
Various mappings, mainly entity

### PR DESCRIPTION
**Pull request by YanisBft**
Entity mappings are copied from #514, closed because outdated.
Several classes are now sorted in the right folders
All fields in GameOption are now mapped

### CODE REVIEW

**Review by modmuss50 — APPROVED** [review]

File: mappings/net/minecraft/sortme/OptionMainHand.mapping [hunk 1]
```diff
@@ -1,3 +0,0 @@
-CLASS aie net/minecraft/sortme/OptionMainHand
```
Comment (liach) [inline mappings/net/minecraft/sortme/OptionMainHand.mapping:1]: Where did this go?

Comment (TehNut) [reply mappings/net/minecraft/sortme/OptionMainHand.mapping:1]: [`net/minecraft/client/options/MainHandOption`](https://github.com/FabricMC/yarn/pull/524/files#diff-bec61871facd317449b6b72fcf77d368)

**Review by raphydaphy — APPROVED** [review]

File: mappings/net/minecraft/client/options/MainHandOption.mapping [hunk 2]
```diff
@@ -0,0 +1,3 @@
+CLASS aie net/minecraft/client/options/MainHandOption
```
Comment (Gegy) [inline mappings/net/minecraft/client/options/MainHandOption.mapping]: Git's not detecting it as a conflict, but this has been mapped to `net/minecraft/util/AbsoluteHand`

Comment (YanisBft) [reply mappings/net/minecraft/client/options/MainHandOption.mapping]: Resolved, though it's not fair that I have to resolve conflicts while I've done the pr before. But whatever ^^

**Review by Gegy — APPROVED** [review]

**Pull request merged**